packages/cli/src/WorkflowHelpers.ts

```diff
@@ -524,3 +527,41 @@ export type NameRequest = Express.Request & {
 		name?: string;
 	};
 };
+
+/**
+ * Build a `where` clause for a `find()` or `findOne()` operation
+ * in the `shared_workflow` or `shared_credentials` tables.
+ */
+export function whereClause({
```

Comment:
I think this function should be moved to `UserManagementHelpers` - we might decide to move them somewhere else later, but I don't think this is the correct place.

Reply:
As discussed, to be centralized once other parallel PRs are merged.